www/js/search.js

```diff
@@ -740,7 +753,8 @@ module.exports = {
     // load verses for bani based on baniID and the length that user has decided
     banidb.loadBani(BaniID, baniLengthCols[baniLength]).then(rowsDb => {
       // create a unique shabadID for whole bani, and append it with length
-      const shabadID = `${rowsDb[0].Token || rowsDb[0].Bani.Token}-${baniLength}`;
+      const shabadID = `${rowsDb[0].Token || rowsDb[0].Bani.Token}-${baniLength}-${rowsDb[0].ID ||
```

Comment:
@inderpreetsingh please review if this is fine. needed to get the bani id on line selection.